### PR TITLE
Or 1504 update immutables.go depends on the binding files generated

### PR DESCRIPTION
**Pull request by AaronLee22**
**Added processes for binding UniswapV3 compiled with Hardhat.**
- As we all know, Optimism compiles all the contracts with Foundry, but UniswapV3 is compiled with Hardhat.
- Therefore, I need to add some code and .sh files that can bind Hardhat-artifacts(UniswapV3 artifacts).

### CODE REVIEW

File: op-bindings/Makefile [hunk 1]
```diff
@@ -3,6 +3,8 @@ SHELL := /usr/bin/env bash
 pkg := bindings
 monorepo-base := $(shell dirname $(realpath .))
 contracts-dir := $(monorepo-base)/packages/tokamak/contracts-bedrock
+hardhat-artifacts-dir := $(monorepo-base)/packages/tokamak/contracts-bedrock
```
Comment (ohbyeongmin) [inline op-bindings/Makefile:6]: this is same with `contracts-dir`

File: op-bindings/predeploys/addresses.go [hunk 2]
```diff
@@ -95,6 +113,15 @@ func init() {
 	Predeploys["SchemaRegistry"] = &SchemaRegistryAddr
 	Predeploys["EAS"] = &EASAddr
 	Predeploys["WETH"] = &WETHAddr
+	Predeploys["Permit2"] = &Permit2Addr
+	Predeploys["QuoterV2"] = &QuoterV2Addr
+	Predeploys["SwapRouter02"] = &SwapRouter02Addr
```
Comment (ohbyeongmin) [inline op-bindings/predeploys/addresses.go:118]: could you delete this space?

File: op-bindings/Makefile (hunk 1 above)
Comment (AaronLee22) [reply op-bindings/Makefile:6]: You're right! Let me fix it right away!

**Review by rlgns98kr — APPROVED** [review]
Thank you very much! 🙇🏼‍♂️

**Pull request merged**